efel/tests/test_basic.py

```diff
@@ -1967,6 +1968,35 @@ def test_decay_time_constant_after_stim2():
         feature_values['decay_time_constant_after_stim'][0], rtol=0, atol=1e-1)
 
 
+def test_multiple_decay_time_constant_after_stim():
+    """basic: Test multiple_decay_time_constant_after_stim"""
```

Comment:
Do you have a trace with multiple steps for this test? It would make more sense in my opinion

Reply:
I am not sure I want to add new data to the package.